cluster/gce/config-common.sh

```diff
@@ -114,3 +124,15 @@ function get-alias-range-size() {
 # NOTE: Avoid giving nodes empty scopes, because kubelet needs a service account
 # in order to initialize properly.
 NODE_SCOPES="${NODE_SCOPES:-monitoring,logging-write,storage-ro}"
+
```

Comment:
Can you add comments here noting what these are for.

Reply:
Done.